.github/workflows/title.yml

```diff
@@ -18,7 +31,17 @@ jobs:
           PR_NUMBER: ${{ github.event.pull_request.number }}
           PR_TITLE: ${{ github.event.pull_request.title }}
         run: |
-          gh pr edit $PR_NUMBER --title "Build: $PR_TITLE"
```

Comment:
https://github.com/content-services/content-sources-frontend/commit/4086622a1abc0d6aae31818e248dcbeb4a47f651

So this patch was not working? Do you have some log where this is failing?

Reply:
So it shows green - it failed because it was missing gh command?